src/Network/Network/help/New-AzVpnSiteLinkConnection.md

```diff
@@ -0,0 +1,235 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```

Comment:
Missing url, should be `https://docs.microsoft.com/en-us/powershell/module/az.network/new-azvpnsitelinkconnection`

Reply:
thanks. done